.htmllintrc

```diff
@@ -0,0 +1,15 @@
+{
+ "attr-name-ignore-regex": false,
+ "attr-name-style":["lowercase"],
```

Comment:
Actually it is there https://github.com/htmllint/htmllint/wiki/Options
but its not `kebab-case`, its `dash`